impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -47,6 +52,7 @@ public class InVMKafkaCluster implements KafkaCluster {
     private final ZooKeeperServer zooServer;
     private final List<Server> servers;
     private final List<String> bootstraps = new ArrayList<>();
+    private final Supplier<KafkaClusterConfig.KafkaEndpoints> kafkaEndpointsSupplier;
```

Comment:
I'm not entirely convinced this is the right place to keep this. 

It should maybe be stored in the clusterConfig and accessed via there.